integrationTests/mock/scQueryMock.go

```diff
@@ -1,13 +1,17 @@
 package mock
 
 import (
+	"math/big"
+
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
 	"github.com/ElrondNetwork/elrond-go/process"
 	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
 )
 
 // ScQueryMock -
 type ScQueryMock struct {
```

Comment:
this is a stub, not a mock. please rename

Reply:
done.